bin/daemon.php

```diff
@@ -63,7 +64,9 @@
 
 DI::init($dice);
 \Friendica\Core\Logger\Handler\ErrorHandler::register($dice->create(\Psr\Log\LoggerInterface::class));
-$a = DI::app();
+
+// Check the database structure and possibly fixes it
+Update::check(DI::basePath(), true, DI::mode());
```

Comment:
That check will only be done on restarting the daemon, If you want to enable the check during normal processing, you have to place it inside the loop, possibly at the place where the cron call is executed.

Reply:
Because there's already a check inside the worker itself, we don't need it at the daemon too .. Seems redundant

Reply:
What worker? `bin/worker.php`? It isn't used by the daemon.

Reply:
oh, my mistake sry